app/src/androidTest/java/net/skyscanner/backpack/compose/button/BpkButtonTest.kt

```diff
@@ -96,7 +99,7 @@ class BpkButtonTest(flavour: Flavor) : BpkSnapshotTest() {
     // icon is bigger on large size, so we need to test this
 
     capture {
-      BpkButton("Button", icon, BpkButtonIconPosition.Start, type = type, size = size, onClick = {})
+      BpkButton("Button", icon, BpkButtonIconPosition.End, type = type, size = size, onClick = {})
```

Comment:
why was this changed to end?

Reply:
ah you're right! i got confused by the github formatting as it didn't include the function title